csharp/test/Ice/operations/MyDerivedClassAMDI.cs

```diff
@@ -736,6 +736,12 @@ public ValueTask opOnewayAsync(Current current)
             throw new Test.SomeException();
         }
 
+        public ValueTask opOnewayMetadataAsync(Current current)
+        {
+            // "return" exception when called two-way, otherwise succeeds.
+            throw new Test.SomeException();
+        }
+
```

Comment:
```suggestion
        // "return" exception when called two-way, otherwise succeeds.
        public ValueTask opOnewayMetadataAsync(Current current) => throw new Test.SomeException();
```